examples/postgresql/assembly/index.ts

```diff
@@ -0,0 +1,34 @@
+import { postgresql } from "@hypermode/functions-as";
+
+export function queryPeople(): string {
+  return postgresql.query("neon", "select * from people", []);
+}
```

Comment:
```suggestion
export function queryPeople(): Person[] {
  return postgresql.query<Person>("neon", "select * from people");
}
```

(see feedback on `getPerson` example)